src/styles/index.scss

```diff
@@ -2,2 +1,244 @@
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav-links {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  &__nav-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    line-height: 60px;
+  }
+
+  &__nav-item {
+    display: inline-block;
+    margin-left: 17px;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__nav-item:first-child {
+    margin-left: 0;
+  }
+
+  &__nav-link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
+  }
+
+  &__nav-link:hover {
+    color: $blue-accent;
+  }
+
+  &__nav-link--is-active {
+    color: $blue-accent;
+    position: relative;
+  }
+
+  &__nav-link--is-active::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: $blue-accent;
+    border-radius: 2px;
+  }
+
+  &__product-card {
+    display: grid;
+    grid-template-columns: repeat(1, 200px);
+    grid-template-rows: repeat(1, 408px);
+    gap: 46px 48px;
+
+    grid-auto-flow: row;
+    place-content: center;
+
+    padding: 50px 40px;
+  }
+
+  &__element {
+    box-sizing: border-box;
+    width: 200px;
+    height: 408px;
+    border: 1px solid #f3f3f3;
+    background: $white-accent;
+    border-radius: 5px;
+    transition: transform 0.3s ease;
+  }
+
+  &__element:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__info {
+    margin: 0 16px 16px;
+    display: flex;
+    flex-direction: column;
+    width: 166px;
+  }
+
+  &__name {
+    margin: 0;
+    line-height: 18px;
+    margin-bottom: 4px;
+    cursor: pointer;
+  }
+
+  &__name:hover {
+    color: $card-title-color;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 6px;
+  }
+
+  &__rating {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    line-height: 16px;
+    margin-bottom: 14px;
+    width: 168px;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    margin-left: 19px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 18px;
+    margin-bottom: 16px;
+    width: 168px;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-weight: 400;
+    margin-right: 85px;
```

Comment:
The right margin of '85px' seems to be a magic number. Magic numbers in the code can lead to confusion and difficulties when maintaining the code. It's better to use variables or calculate spacings based on a consistent scale or grid.